include/zenoh-pico/config.h

```diff
@@ -45,6 +45,7 @@
 #define Z_FEATURE_LOCAL_SUBSCRIBER 0
 #define Z_FEATURE_PUBLISHER_SESSION_CHECK 1
 #define Z_FEATURE_BATCHING 1
+#define Z_FEATURE_MEMOIZATION 0
```

Comment:
```suggestion
#define Z_FEATURE_MEMORIZATION 0
```

Reply:
The technical term is Memoization though, https://en.wikipedia.org/wiki/Memoization. But yeah that might not be transparent to the user, in that case we might as well go to something like `Z_FEATURE_RX_CACHE`